src/index.tsx

```diff
@@ -2,16 +2,15 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App";
-import { BrowserRouter } from "react-router-dom";
+import { HashRouter } from "react-router-dom";
 
-const root = ReactDOM.createRoot(
-  document.getElementById("root") as HTMLElement,
-);
+const GH_REPOSITORY = process.env.REACT_APP_GH_REPOSITORY;
```

Comment:
Is line 7 necessary? If we leave this blank what are the effects? Since we need to set the `homepage` value in `package.json` anyway I'm just thinking whether this is a necessary addition?

This is more out of curiosity, I don't feel too strongly either way.

Reply:
Yeah IIRC the router needs to know what it should expect as the base for the application, apparently it doesn't use the package.json data

Reply:
Interesting, that's not how I remember it but it's been a while since I last used it.